lib/utils/exceptions.dart

```diff
@@ -0,0 +1,13 @@
+String extractExceptionMessage(Object exception) {
+  final detailRegex = r'((?<=\"detail\":\")(.*)(?=.*\"}))';
+  final nsLocalizedRegex = r'((?<={NSLocalizedDescription=)(.*)(?=}))';
+  final lnUrlRegex = r'((?<=LNURL:.)(.*)(?=.method:))';
+  final grouped = RegExp('$nsLocalizedRegex|$lnUrlRegex|$detailRegex');
+
+  final match = grouped.stringMatch(exception.toString());
+  if (match != null) {
+    return match;
+  }
+
+  return exception.toString();
```

Comment:
We can make use of null-aware operator here:
return grouped.stringMatch(exception.toString()) ?? exception.toString();

Reply:
🤯 applied